docs/README.rst

```diff
@@ -244,19 +242,58 @@ distributed Python. You can think of it as a ``trio``
 stdlib's ``multiprocessing`` but built on async programming primitives
 from the ground up.
 
-``tractor``'s nurseries let you spawn ``trio`` *"actors"*: new Python
-processes which each run a ``trio`` scheduled runtime - a call to ``trio.run()``.
-
 Don't be scared off by this description. ``tractor`` **is just** ``trio``
 but with nurseries for process management and cancel-able streaming IPC.
 If you understand how to work with ``trio``, ``tractor`` will give you
-the parallelism you've been missing.
+the parallelism you may have been needing.
+
+
+Wait, huh?! I thought "actors" have messages, and mailboxes and stuff?!
+***********************************************************************
+Let's stop and ask how many canon actor model papers have you actually read?
+
+From the author's mouth, **the only thing required** is `adherance to`_
+the `3 axioms`_, *and that's it*.
+
+To get more fired up on the matter, please read these issues:
+
+- https://github.com/goodboy/tractor/issues/210
+- https://github.com/goodboy/tractor/issues/18
 
-"Actors" communicate by exchanging asynchronous messages_ and avoid
-sharing state. The intention of this model is to allow for highly
-distributed software that, through the adherence to *structured
-concurrency*, results in systems which fail in predictable and
-recoverable ways.
+*News flash*: many "actor systems" aren't really "actor models" since
+they **don't adhere** to the `3 axioms`_ and pay even less attention to 
+the problem of *unbounded non-determinism* (which was the whole point
+for creation of the model in the first place).
+
+Again, ``tractor`` does seem to adhere to the base requirements of
```

Comment:
I would remove the `Again` & `does seem to` parts and replace them with a more "certain / to the point"? statement:

> ``tractor`` aims to adhere to the base requirements of an "actor model".